src/Serializers/FingerprintSerializer.php

```diff
@@ -48,15 +47,11 @@ public function addIdToSerialization( Entity $entity, array &$serialization ) {
 		$serialization['id'] = $id->getSerialization();
 	}
 
-	public function addLabelsToSerialization( Entity $entity, array &$serialization ) {
-		$labels = $entity->getFingerprint()->getLabels();
-
+	public function addLabelsToSerialization( TermList $labels, array &$serialization ) {
```

Comment:
Yeah... I'm quite unhappy about what that code looked like, this commit only fixing some of it. Too bad it got through review somehow